### PR TITLE
[toypics] fix extractor

### PR DESCRIPTION
**Pull request by vobe**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Toypics support was no longer working, this commit makes the unit tests pass again

### CODE REVIEW

File: youtube_dl/extractor/toypics.py [hunk 1]
```diff
@@ -25,11 +25,11 @@ def _real_extract(self, url):
         video_id = mobj.group('id')
         page = self._download_webpage(url, video_id)
         video_url = self._html_search_regex(
-            r'src:\s+"(http://static[0-9]+\.toypics\.net/flvideo/[^"]+)"', page, 'video URL')
+            r'src="(https?://static[0-9]+\.toypics\.net/flvideo/[^"]+)"', page, 'video URL')
```
Comment (dstftw) [inline youtube_dl/extractor/toypics.py]: `_parse_html5_media_entries`.

File: youtube_dl/extractor/toypics.py [hunk 2]
```diff
@@ -25,11 +25,11 @@ def _real_extract(self, url):
...
         title = self._html_search_regex(
-            r'<title>Toypics - ([^<]+)</title>', page, 'title')
+            r'<h1 class="view-video-title">([^<]+)</h1>', page, 'title')
```
Comment (dstftw) [inline youtube_dl/extractor/toypics.py]: Make more relaxed and add title regex as fallback.

File: youtube_dl/extractor/toypics.py [hunk 3]
```diff
@@ -71,8 +71,8 @@ def _real_extract(self, url):
                 note='Downloading page %d/%d' % (n, page_count))
             urls.extend(
                 re.findall(
-                    r'<p class="video-entry-title">\s+<a href="(https?://videos.toypics.net/view/[^"]+)">',
-                    lpage))
+                    r'<div class="preview">\s+<a href="(https?://videos.toypics.net/view/[^"]+)">',
+                    lpage, re.S+re.M))
```
Comment (dstftw) [inline youtube_dl/extractor/toypics.py]: Move flags into regex.

**Comment by vobe** [comment]
I corrected my changes according to your feedback and rebased

**Comment by dstftw** [comment]
Do you want to be credited as one of the authors of youtube-dl? If so under what name?

**Comment by vobe** [comment]
I don't wish to be credited for such a small fix. Thanks for merging!

**Pull request merged**